Lil-Peep/src/styles/modules/site-collection.scss

```diff
@@ -219,26 +219,18 @@
     background-color: #{$view-green};
     border-radius: 3px;
     border-style: none;
-    padding: 10px 0;
+    color: #{$color-body};
+    display: block;
+    font-family: 'Oswald', sans-serif;
+    font-weight: bold;
+    padding: 10px 15px;
     text-align: center;
-    width: 100%;
+    text-decoration: none;
 
     &:hover {
       background-color: #{$on-hover-button};
```

Comment:
You don't need to use the #{} for all the variables  just when it's a breakpoint